src/components/VideoPlayer.vue

```diff
@@ -1,105 +1,106 @@
 <template>
-  <div id="app">
-  <div class="controls">
-    <input v-model="videoId">
-    <button @click="stopVideo">stop</button>
-    <button @click="playVideo">Play</button><br />
-    <input type="range" min="0" :max="duration" step="0.01" v-model="loopStart"><input v-model="loopStart"><br />
-    <input type="range" min="0" :max="duration" step="0.01" v-model="loopEnd"><input v-model="loopEnd"><br />
-  </div>
-  
-  <div id="player"></div>
-  
-  <div class="info">
-    Video: <span v-text="videoId"></span><br />
-    Duration: <span v-text="duration"></span><br />
-    Current Time: <span v-text="currentTime"></span><br />
-    Loop Seconds: <span v-text="loopStartSeconds"></span> <span v-text="loopEndSeconds"></span><br />
-  </div>
-</div>
+    <div id="app">
+        <div class="controls">
+            <input v-model="videoId">
+            <button @click="stopVideo">stop</button>
+            <button @click="playVideo">Play</button><br />
+            <input type="range" min="0" :max="duration" step="0.01" v-model="loopStart"><input v-model="loopStart"><br />
+            <input type="range" min="0" :max="duration" step="0.01" v-model="loopEnd"><input v-model="loopEnd"><br />
+        </div>
+
+        <div :id="playerId"></div>
+
+        <div class="info">
+            Video: <span v-text="videoId"></span><br />
+            Duration: <span v-text="duration"></span><br />
+            Current Time: <span v-text="currentTime"></span><br />
+            Loop Seconds: <span v-text="loopStartSeconds"></span> <span v-text="loopEndSeconds"></span><br />
+        </div>
+    </div>
 </template>
+
 <script>
 /* eslint-disable */
-    export default {
-        props: [this.comboId],
```

Comment:
using this.comboId would not work in this context and this would be undefined. Here you're essentially setting property names, and then what type of data you expect them to be.